application/src/main/java/org/togetherjava/tjbot/commands/ReloadCommand.java

```diff
@@ -0,0 +1,137 @@
+import net.dv8tion.jda.api.utils.cache.SnowflakeCacheView;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public record ReloadCommand(CommandHandler commandHandler) implements Command {
+
+    @Override
+    public @NotNull String getCommandName() {
+        return "reload";
+    }
+
+    @Override
+    public @NotNull String getDescription() {
+        return "Uploads all existing slash-commands to Discord so they're fully up-to-date.";
+    }
+
+    @Override
+    public boolean isGuildOnly() {
+        return true;
+    }
+
+    @Override
+    public void onSlashCommand(SlashCommandEvent event) {
+        Member member = event.getMember();
+
+        if (member.hasPermission(Permission.MANAGE_SERVER)) {
+            event.reply(
+                    "You sure? You can only reload commands a few times each day, so don't overdo this.")
+                .addActionRow(
+                        Button.of(ButtonStyle.SUCCESS, generateComponentId(member.getId()),
+                                "Affirmative!"),
+                        Button.of(ButtonStyle.DANGER, generateComponentId(member.getId()),
+                                "Wrong command! Sorry"))
+                .queue();
+        } else {
+            event.reply("You need the MANAGE_SERVER permission to use this command!")
+                .setEphemeral(true)
+                .queue();
+        }
+    }
+
+    @Override
+    public void onButtonClick(ButtonClickEvent event, List<String> idArgs) {
+        Member member = event.getMember();
+
+        if (member.getId().equals(idArgs.get(0))) {
+            switch (event.getButton().getStyle()) {
+                case DANGER -> {
+                    event.reply("Next time use the right command!").queue();
+                    event.getMessage().editMessageComponents().queue();
+                }
+
+                case SUCCESS -> {
+                    event.deferReply().queue();
+                    List<Command> commands = commandHandler.getCommandList();
+                    List<CommandListUpdateAction> restActions;
+
+                    // * loads all RestActions from updating the Guild and Global commands
+                    restActions = getGuildCommandUpdateRestActions(event, commands);
+                    restActions.add(getGlobalCommandUpdateRestAction(event, commands));
+
+                    // * Triggers all RestActions, when they're all finished the message gets send.
+                    RestAction.allOf(restActions).queue(updatedCommands -> {
+                        event.getHook()
+                            .editOriginal(
+                                    "Commands successfully reloaded! *Global commands can take upto 1 hour to load")
+                            .queue();
+                    });
+                }
+            }
+        }
+    }
+
+    public CommandListUpdateAction getGlobalCommandUpdateRestAction(Event event,
```

Comment:
same as above

Reply:
While the methods were public, they aren't for use, it's a command not something public.
So I don't see any points in making docs, the code itself is extremely simple,
It feels like creating docs for methods like this is just a waste of everybody's time